src/engine/Server.cpp

```diff
@@ -62,17 +71,15 @@ void Server::run() {
       << std::endl;
 
     ad_utility::Socket client;
+    // Concurrent accept used to be problematic because of the Thundering Herd
+    // problem but this should be fixed on modern OSs
     bool success = _serverSocket.acceptClient(&client);
     if (!success) {
-      LOG(ERROR) << "Socket error while trying to accept client" << std::endl;
+      LOG(ERROR) << "Socket error in acceot" << std::strerror(errno) << std::endl;
       continue;
     }
```

Comment:
As far as I can tell, the setKeppAlive(true) went missing here. Is that intended?